commons/src/main/java/io/hyscale/commons/logger/WorkflowLogger.java

```diff
@@ -221,5 +221,9 @@ public static void logPersistedActivities() {
         }
         persistedActivities.clear();
     }
-    
+
+    public static void write(String message) {
```

Comment:
Method name is out of sync I feel!
What do you think about this @tushar-tushar ?

Reply:
I can rename it as log keeping it in sync with other method names?